docs/create_your_own_extension_layer.md

```diff
@@ -38,13 +38,22 @@ RUN make -j `nproc` && make install
 
 RUN cp `php-config --extension-dir`/pgsql.so /tmp/pgsql.so
 ```
-Some env variables are set and can be used in this environment. `PHP_BUILD_DIR` is `/tmp/build/php`, `INSTALL_DIR` is `/opt/bref`.
+
+Compile the extension here.
+In some cases, download the source code, install the libraries required for compilation, perform pecl install, etc, describe the processing to create all files required for extension.
```

Comment:
> describe the processing to create all files required for extension.

Is this a comment to yourself? =)

Reply:
hmm.. What I mean is that all necessary processing needs to be written here.
I'm sorry my English isn't very good.

Reply:
I removed that sentence. How is this? :)